charts/internal/aws-infra/templates/main.tf

```
@@ -24,6 +24,13 @@ resource "aws_vpc" "vpc" {
 {{ include "aws-infra.common-tags" .Values | indent 2 }}
 }
 
+{{ range $ep := .Values.vpc.gatewayEndpoints }}
+resource "aws_vpc_endpoint" "vpc_gwep_{{ $p }}" {
```

maybe you want 
```suggestion
resource "aws_vpc_endpoint" "vpc_gwep_{{ $ep }}" {
```
?

---

correct, was a typo, thanks!